accounts/views.py

```diff
@@ -47,6 +47,82 @@ def get(self, request):
         return HttpResponseServerError()
 
 
+class DevLoginView(View):
+    """
+    Log in a test user.
+    Does not use Shibboleth
+    """
+
+    @staticmethod
+    def populate_user_data():
```

Comment:
We should add a management script to create a bunch of users with additional useful information (interesting student fields, verified methods of contact, etc). [OHQ example](https://github.com/pennlabs/office-hours-queue/blob/master/backend/ohq/management/commands/populate.py)